libraries/stdlib/src/kotlin/text/Indent.kt

```diff
@@ -14,7 +14,8 @@ package kotlin.text
  *
  * Doesn't affect a line if it doesn't contain [marginPrefix] except the first and the last blank lines.
  *
- * Doesn't preserve the original line endings.
+ * Doesn't preserve the original line endings: any platform-specific line endings (such as `\r\n` on Windows) will be
+ * converted to `\n`.
```

Comment:
Let's drop this wording ("Doesn't preserve the original line endings") since it's been proved to be confusing. It will be enough to describe what actually happens with them.

Reply:
```
The lines in the original string can be separated with `\r\n` (CRLF), `\n` (LF), or `\r` (CR) characters, 
however, the lines in the resulting string will be separated solely with `\n` (LF) character.
```

Reply:
Applied this suggestion, thank you!